### PR TITLE
Updating readme to refer to new style

### PR DESCRIPTION
**Pull request by Kusold**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +23,18 @@ NPM module that wraps the habitrpg-api in an easy to use format.
 var habitapi = require('habit-rpg');
 new habitapi(userId, apiKey);
 ```
+Alternatively, to point at a custom server:
```
Comment (crookedneighbor) [inline README.md]: Was thinking about this today.

:+1: :+1: :+1: :+1: :+1:

**Comment by Kusold** [comment]
I was in here because I wanted to see if I could handle the setting of `dateCompleted` automatically for todos. I think it might be best left for the actual app unfortunately.

**Comment by crookedneighbor** [comment]
@Kusold I agree.

**Comment by crookedneighbor** [comment]
Except for the "unfortunately" part. :sweat_smile:

**Pull request merged**